requirements/base.txt

```diff
@@ -0,0 +1,8 @@
+# Requirements for app run
+
+git+https://github.com/edx/xblock-utils.git@v1.1.1#egg=xblock-utils==1.1.1
+git+https://github.com/edx/i18n-tools.git
+django-statici18n==1.8.0
+edx-i18n-tools==0.4.7
+Mako==1.0.7
```

Comment:
This is not needed so you can remove it.

Reply:
We need this to prevent CI from crashing